### ISSUE TITLE
play-services-games:22.0.0 - Random (Rare) Freeze "com.google.android.gms is currently frozen"

### ISSUE THREAD

**Issue by danoli3** [playgameservices/android-basic-samples#341]
Blocked Rendering for some reason maybe crashed the thread

looking into more detail

Might advise not deploying with 22.0.0 till discovered more 

```
2021-12-17 20:32:19.417 4016-4060/? E/ActivityManager: Failure starting process com.google.android.gms.persistent
    java.lang.SecurityException: Package com.google.android.gms is currently frozen!
        at com.android.server.pm.PackageManagerService.checkPackageStartable(PackageManagerService.java:5570)
        at com.android.server.am.ProcessList.startProcessLocked(ProcessList.java:2102)
        at com.android.server.am.ProcessList.startProcessLocked(ProcessList.java:2774)
        at com.android.server.am.ProcessList.startProcessLocked(ProcessList.java:3003)
        at com.android.server.am.ActivityManagerService.startProcessLocked(ActivityManagerService.java:3953)
        at com.android.server.am.ActiveServices.bringUpServiceLocked(ActiveServices.java:3272)
        at com.android.server.am.ActiveServices.bindServiceLocked(ActiveServices.java:2292)
        at com.android.server.am.ActivityManagerService.bindIsolatedService(ActivityManagerService.java:18123)
        at android.app.ContextImpl.bindServiceCommon(ContextImpl.java:1913)
        at android.app.ContextImpl.bindServiceAsUser(ContextImpl.java:1863)
        at com.android.server.ServiceWatcher.rebind(ServiceWatcher.java:339)
        at com.android.server.ServiceWatcher.onBestServiceChanged(ServiceWatcher.java:310)
        at com.android.server.ServiceWatcher.onBindingDied(ServiceWatcher.java:391)
        at android.app.LoadedApk$ServiceDispatcher.doConnected(LoadedApk.java:2080)
        at android.app.LoadedApk$ServiceDispatcher$RunConnection.run(LoadedApk.java:2116)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:246)
        at android.os.HandlerThread.run(HandlerThread.java:67)
        at com.android.server.ServiceThread.run(ServiceThread.java:44)
```
        

```
    implementation 'androidx.appcompat:appcompat:1.4.0'
    implementation 'androidx.constraintlayout:constraintlayout:2.1.2'
    implementation 'com.android.support.constraint:constraint-layout:2.0.4'
    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
    implementation "com.google.android.gms:play-services-games:22.0.0"
    implementation "com.google.android.gms:play-services-auth:20.0.0"
    implementation "com.google.android.gms:play-services-base:18.0.0"
    implementation "com.google.android.gms:play-services-basement:18.0.0"
    implementation "com.google.android.gms:play-services-instantapps:18.0.0"
    implementation "com.google.android.gms:play-services-appset:16.0.1"
    ```

**Comment by danoli3** [comment]
Difficult replicating, was on a hardware device. Have not been able to replicate thus far

**Comment by danoli3** [comment]
Potentially fixed in 20.0.1
